src/react/account/AccountContent.tsx

```diff
@@ -17,16 +17,16 @@ import AccountUpdateUser from './AccountUpdateUser';
 import AccountUserAccessKeys from './AccountUserAccessKeys';
 import CreateWorkflow from '../workflow/CreateWorkflow';
 import Workflows from '../workflow/Workflows';
+import { useAccounts } from '../utils/hooks';
```

Comment:
I think we can remove both useSelector and useAccounts imports because we're not using them anymore